test/common/http/conn_pool_grid_test.cc

```diff
@@ -718,6 +723,11 @@ TEST_F(ConnectivityGridTest, ConnectionCloseDuringCreation) {
   ASSERT_TRUE(optional_it1.has_value());
   EXPECT_EQ("HTTP/3", (**optional_it1)->protocolDescription());
 
+  TestScopedRuntime scoped_runtime;
+  Runtime::LoaderSingleton::getExisting()->mergeValues(
+      {{"envoy.reloadable_features.postpone_h3_client_connect_till_enlisted",
+        (GetParam() ? "true" : "false")}});
```

Comment:
Is this the only test that uses the new parameterization? If so, I wonder if we can create a new test fixture for this class and just parameterize that to avoid running all the other tests twice.

Reply:
done